docs/website/content/v0.4/en/guides/getting-started/firecracker.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Firecracker
+---
+
+In this guide we will create a Kubernetes cluster using Firecracker.
+
+## Requirements
+
+- KVM
+- firecracker (v0.21.0 or higher)
+- CNI installed to `/opt/cni` (`CAP_SYS_ADMIN` and `CAP_NET_ADMIN` capabilities are required)
```

Comment:
we need specific CNI modules:   https://github.com/talos-systems/talos/blob/master/internal/pkg/provision/providers/firecracker/README.md